app/models/pin.rb

```diff
@@ -24,34 +24,51 @@ class Pin < ActiveRecord::Base
   # we have so few records we don't really need multiple shards
   settings index: { number_of_shards: 1 } do
     mapping do
-      # TODO
-      indexes :surgeon
-      indexes :procedure
-      indexes :pin_images
-
-      indexes :description, type: 'text' do
-        indexes :description, analyzer: 'snowball'
-        indexes :tokenized, analyzer: 'simple'
+      # no need to do full text search on this
+      indexes :state, type: 'keyword'
+      # keyword (exact term) doesn't make sense for these
+      indexes :description, type: 'text', analyzer: 'english'
+      indexes :details, type: 'text', analyzer: 'english'
+
+      # we use the english analyzer, and do keywords as well as text
+      # because then we can do significant terms aggregations
+      # https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html#english-analyzer
+      indexes :complications do
```

Comment:
Eventually would be cool to put phonetic in as an analyzer given the plugin is on Bonsai and it might help with some queries where people are challenged to spell right.